src/domain/breed.domain.ts

```diff
@@ -0,0 +1,23 @@
+export interface Breeds {
+  breeds: string[];
+}
+// object definition Lambda response
+export interface GetBreedsResponse {
+  statusCode: number;
+  body: Breeds;
+  message?: string;
+}
+// object definition from API response
+export interface BreedsApiResponse {
+  message: Message;
+  status: string;
+  code?: number;
+}
+
+export interface Message {
```

Comment:
Is it possible that any of these interfaces are generic enough to be shared across multiple lambdas?